### ISSUE TITLE
ReportToManagerEnabled and ReportToOriginatorEnabled are not entirely accurate

### ISSUE THREAD

**Issue by VShrubowich** [MicrosoftDocs/office-docs-powershell#3597]
In Exchange Online, and I believe most if not all of the Exchange Server versions listed, when ReportToManagerEnabled is set to $true, delivery status notifications are sent to *one* owner of the group, not all of the owners.  (The owner that gets the DSN appears to be the one that is first in the "ManagedBy" property of the group, which can change as owners are added and removed and doesn't depend on alphabetical or any other order I can determine.)
When ReportToOriginatorEnabled is $true, DSNs are sent to senders who send messages *as* this group (not *to* this group).  The person who sent the message as the group receives the DSN.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 8f2a78c2-a2e3-ce26-7ffb-2caa9a6622a3
* Version Independent ID: 2a579c6c-8a7d-df6f-dd27-b373fee14ee1
* Content: [Set-DistributionGroup (exchange)](https://docs.microsoft.com/en-us/powershell/module/exchange/users-and-groups/set-distributiongroup?view=exchange-ps#feedback)
* Content Source: [exchange/exchange-ps/exchange/users-and-groups/Set-DistributionGroup.md](https://github.com/MicrosoftDocs/office-docs-powershell/blob/master/exchange/exchange-ps/exchange/users-and-groups/Set-DistributionGroup.md)
* Service: **exchange-powershell**
* GitHub Login: @chrisda
* Microsoft Alias: **chrisda**

**Comment by yogkumgit** [comment]
@officedocsbot assign @yogkumgit

**Comment by dariomws306** [comment]
Hi @VShrubowich, thank you for your feedback.

ReportToManagerEnabled send emails to the user you can find on the managedBy Active Directory property. You can see this on the description of this parameter. It's not a multivalue property:
https://docs.microsoft.com/en-us/windows/desktop/adschema/a-managedby

The ReportToOriginatorEnabled parameter specifies whether delivery status notifications are sent **to senders who send messages to this group**.

**Comment by yogkumgit** [comment]
@dariomws306  Thank you very much for the contribution and sharing this explanation. @VShrubowich 
Hope this comment is helpful for you. Thanks for taking out some time to open the issue. Appreciate and encourage you to do the same in future also.

**Comment by yogkumgit** [comment]
@officedocsbot close

**Comment by rbaxe** [comment]
The documentation doesn't seem to reflect what I'm seeing in practice. I have a group which I want to restrict sending to and also not send delivery failures when people send to it. I set AcceptMessagesOnlyFrom  to a single user and then set ReportToOriginatorEnabled to false. By my reading of the documentation an if another user who was not listed in AcceptMessagesOnlyFrom would generate a DSN however the DSN should be suppressed by the ReportToOriginatorEnabled being set to false. This is not what happens. Users sending to the group still receive DSNs. Note I'm working in EXO.

**Comment by dariomws** [comment]
Hi @rbaxe, thank you for your feedback.

Please have in mind that it could take some time to apply and also the following sentence:
"The ReportToManagerEnabled and ReportToOriginatorEnabled parameters affect the return path for messages sent to the group. Some email servers reject messages that don't have a return path. Therefore, you should set one parameter to $false and one to $true, but not both to $false or both to $true."

**Comment by rbaxe** [comment]
Thanks @dariomws! In terms of waiting for the setting to apply I have waited over 24 hours and the groups still seem to generate DSN's when users who are not allowed to send them emails send emails to the group. The goal is to literally never generate a DSN when someone sends to the group that wasn't supposed to. Is that the expected behavior? The sentence you added regarding the return path would indicate that we are looking at behavior after the group is expanded and resolved. I'm interested in suppressing the DSN when the email is rejected by the initial Exchange server because the user is not permitted to send mail to that group.

**Comment by dariomws** [comment]
@rbaxe I've verified and I experienced the same behaviour.

For your inquiry I need to kindly suggest opening a support ticket directly from your tenant's administration, they will report to the product group as here we are limited to documentation issues and improvements. Anyway, I'll also send this feedback to them.

Support URL: https://docs.microsoft.com/microsoft-365/admin/contact-support-for-business-products.

Thank you!
